packages/headless/src/features/facets/range-facets/numeric-facet-set/numeric-facet-actions.ts

```diff
@@ -181,6 +184,12 @@ export interface UpdateNumericFacetSortCriterionActionCreatorPayload {
  */
 export const updateNumericFacetSortCriterion = updateRangeFacetSortCriterion;
 
+/** Updates the range algorithm of a numeric facet.
+ * @param facetId (string) The unique identifier of the facet (e.g., `"1"`).
+ * @param rangeAlgorithm (RangeFacetRangeAlgorithm) The target range algorithm.
+ */
```

Comment:
This style of annotation is the old approach which will removed in the near future.

Instead, we should create an interface, and annotate the properties of the interface. Then we would expose the action via the numeric-facet-actions-loader file. Same for the numeric facet.